enigma-js/src/Enigma.js

```diff
@@ -456,6 +456,19 @@ export default class Enigma {
     return emitter;
   }
 
+  /**
+   * Decrypt task result
+   *
+   * @param {Task} task - Task wrapper for contract deployment and regular tasks
+   * @return {Task} Decrypted task result wrapper
+   */
+  async decryptTaskResult(task) {
+    const {privateKey} = this.obtainTaskKeyPair();
```

Comment:
I find this counterintuitive, if this function is returning a pair of values, like here: https://github.com/enigmampc/enigma-contract/blob/9736d628a2ef9a14661b06dfdb6def73c5cd67f6/enigma-js/src/Enigma.js#L113
but you're grabbing only one. Just checking that this is proper coding.... I don't know for sure

Reply:
just syntactic sugar, and avoiding linter complaining about unused `publicKey` var